### PR TITLE
added Safari Pinned tab icon

### PR DESCRIPTION
**Pull request by NathanTheGr8**
added a pure black sag of the sonar logo
make a link in the index file

### CODE REVIEW

**Comment by NathanTheGr8** [comment]
This is for #1122 , Never done a pull request for a big project before. Also I wasn't able to check that it works, I couldn't figure out how to compile it and run as an OX app. It is probably something simple so please post and I will check that it works.

File: src/UI/index.html [hunk 1]
```diff
@@ -29,6 +29,7 @@
     <link rel="apple-touch-icon" sizes="72x72" href="/Content/Images/touch/72.png"/>
     <link rel="apple-touch-icon" sizes="114x114" href="/Content/Images/touch/114.png"/>
     <link rel="apple-touch-icon" sizes="144x144" href="/Content/Images/touch/144.png"/>
+    <link rel="mask-icon" href="/Logo/sonarr_black_and_white.svg" color="#36C6F4">
```
Comment (markus101) [inline src/UI/index.html]: `/Logo` isn't accessible by the web server, it needs to be in `/Content/Images`.

**Comment by markus101** [comment]
Not sure that the outline works for the image. Also the SVG needs a transparent background. I'll take a stab at it later today.

File: src/UI/index.html (hunk 1 above)
Comment (NathanTheGr8) [inline src/UI/index.html]: ok I will change the file location, thanks

**Comment by NathanTheGr8** [comment]
@markus101 I believe the svg does have a transparent background. The problem about making the svg is that apple just wants only black svg. The sonarr logo can be easy changed to black and white, however the white really need to be transparent. I accomplished this by a kind of crude manor of tracing and png output of the original logo in black and white. I added the think circle line because I thought it looked weird otherwise. It you can make a better svg please do, I am pretty bad at using graphic adobe illustrator.

**Comment by markus101** [comment]
After way too much effort:

https://gist.github.com/markus101/8b48c9e2a4de9d598338

I think it looks less wheel like. Unless anyone else has an opinion I think this is what we should go for.

Name is `logo.svg` in the safari folder you created and I think we're set.

**Comment by NathanTheGr8** [comment]
Looks great! I liked the circle but what ever. You were able to do it much better.

**Comment by NathanTheGr8** [comment]
just added some Tile images for #1108

**Comment by markus101** [comment]
> just added some Tile images for #1108

Each PR should really only address one issue unless they are related and separating them is more effort than its worth.

I'm not seeing the icons load when pinning to from Edge in Windows 10, but the title is there. Looks like the sizes recommended are different than I listed previously. The size should match the meta tag name.

```
<meta name="application-name" content="Sonarr"/>
<meta name="msapplication-TileColor" content="#000000"/>
<meta name="msapplication-square70x70logo" content="tiny.png"/>
<meta name="msapplication-square150x150logo" content="square.png"/>
<meta name="msapplication-wide310x150logo" content="wide.png"/>
<meta name="msapplication-square310x310logo" content="large.png"/>
```

**Comment by NathanTheGr8** [comment]
Sorry about that, I corrected it. I had changed the names to reflect the exact size like elsewhere in the project. I didn't realize it mattered.

**Comment by markus101** [comment]
Its not the file names that are wrong, its that they don't match the size specified, I much prefer the resolution in the file name instead of tiny, square, etc.

**Comment by NathanTheGr8** [comment]
I notice that when I was committing it however when I used this generator http://www.buildmypinnedsite.com/en, It made the file sizes you saw. I assumed it has something to do with display scaling that the wanted a bigger picture then the size in the name tag.

**Comment by NathanTheGr8** [comment]
I just removed the microsoft stuff I will add that in another merge request later.

**Comment by markus101** [comment]
Thanks, the safari icon has been merged.

**Pull request closed without merging**